### DISCUSSION TITLE
【deepin_V23_beta3】 【一般】 【DDE】多屏布局未记忆/自动恢复

### DISCUSSION THREAD

**Discussion by ZhongYic00** [linuxdeepin/developer-center#7609] [Troubleshooting | 故障排查]
不确定是否特性被砍掉了
V20下连接不同屏幕时，窗口布局会自动记忆与恢复。
复现步骤：
1. 连接显示器
2. 拖拽窗口到副屏
3. 拔下显示器，副屏上窗口自动回到主屏
4. 再插上显示器
当前表现：
5. 窗口布局不变
预期表现：
5. 此前在副屏的窗口自动回到副屏

**Comment by BLumia** [comment]
cc @meiyixiang @justforlxz
